src/intl/pt.json

```diff
@@ -1112,6 +1107,11 @@
           "description": "PhD Biomedical Sciences - University of Illinois Chicago\nAmbassador - One Young World Nigeria",
           "title": "Executive Clinical Advisor"
         },
+        "cheema": {
+          "name": "Faisal Cheema",
+          "description": "Associate Professor of Biomedical & Clinical Sciences, Tilman J. Fertitta College of Medicine, University of Houston",
+          "title": "Executive Clinical Advisor"
+        },
```

Comment:
Out of curiosity, was there a reason behind moving this section from its previous location?